doc/usage/theming.rst

```diff
@@ -219,9 +219,6 @@ These themes are:
   - **headbgcolor** (CSS color): Background color for headings.
   - **headtextcolor** (CSS color): Text color for headings.
   - **headlinkcolor** (CSS color): Link color for headings.
-  - **codebgcolor** (CSS color): Background color for code blocks.
-  - **codetextcolor** (CSS color): Default text color for code blocks, if not
-    set differently by the highlighting style.
```

Comment:
I don't think this sentence is correct, it seems like the highlighting style is ignored.

Reply:
It turns out that this sentence is indeed correct, but there is one case where it is misleading:

When using the `'none'` language, Pygments doesn't set a text color (but it *does* set a background color).

Therefore, when using a dark background (e.g. `pygments_style = 'monokai'`) and using a dark color for `codetextcolor`, the text on `'none'`-language code blocks will be not readable.

Anyway, if the default value for `codetextcolor` is changed to `'unset'` (as suggested in #7721), this will not be a problem anymore.